hal/riscv64/pmap.c

```diff
@@ -342,14 +347,12 @@ char pmap_marker(page_t *p)
 /* Function initializes low-level page mapping interface */
 void _pmap_init(pmap_t *pmap, void **vstart, void **vend)
 {
-	void *v;
```

Comment:
Move these variables under #if 0 - so we know what type these should be of

Reply:
Done